static_files/main.js

```diff
@@ -0,0 +1,22 @@
+
+const  Menu = document.getElementById("HiddenMenu");
+
+document
+  .querySelector("#ShowMenuButton")
+  .addEventListener("click", MakeVisible);
+
+
+function MakeVisible (){
+
+  if (Menu.classList.contains('HideMenu')) {
```

Comment:
This is a advanced suggestion, do you want to implement right now?
I'll suggest to you, to use a object destructuring...

https://dmitripavlutin.com/javascript-object-destructuring/

what do you think?